docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trellix-mvisio-epo-source.md

```diff
@@ -0,0 +1,122 @@
+---
+id: trellix-mvisio-epo-source
+title: Trellix mVision ePO Source
+sidebar_label: Trellix mVision ePO
+description: Learn how to collect event logs using the Trellix mVision ePO.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/trellix-logo.png')} alt="trellix-logo" width="70" />
+
+Trellix is a cybersecurity company that provides cloud-based security solutions for cybersecurity attacks. It provides hardware, software, and services to investigate cybersecurity attacks, protect against malicious software, and analyze IT security risks.
+
+mVision ePO is a key component of the Trellix Security Management Platform, which provides Unified Management of Endpoint, Network, and Data Security, which reduces Incident Response Times, Strengthen Protection, Simplify Risk and Security Management with Automation, and End-to-end Network Visibility and Security.
+
+## Data Source
+
+The Trellix integration fetches event logs using the [Trellix mVision ePO API](https://developer.manage.trellix.com/mvision/apis/v2-events).
+
+## Set up and configuration
+
+TheTrellix mVision ePO source requires you to provide Client ID, Client Secret, and API key. To obtain these, follow the steps below:
+
+1. Login to the [Trellix Developer Portal](https://developer.manage.trellix.com/).
+1. Go to **Self service > [API Access Management](https://developer.manage.trellix.com/mvision/selfservice/access_manag)**. <br/><img src={useBaseUrl('img/send-data/self-service.png')} alt="self-service.png" style={{border: '1px solid black'}} width="600" />
```

Comment:
```suggestion
1. Go to **Self service** > **[API Access Management](https://developer.manage.trellix.com/mvision/selfservice/access_manag)**. <br/><img src={useBaseUrl('img/send-data/self-service.png')} alt="self-service.png" style={{border: '1px solid black'}} width="600" />
```

Reply:
`https://developer.manage.trellix.com/mvision/selfservice/access_manag`

This looks like an incomplete URL.

Reply:
It's good to write an action button name after any actionable items. Like **Self Service** menu or tab or whatever it is.

Reply:
That is the complete URL, the page works with this URL.